src/main/java/core/basesyntax/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        int fruitsInStorage = Storage.storage.get(fruitTransaction.getFruit());
```

Comment:
There's a potential `NullPointerException` here if `get(fruitTransaction.getFruit())` returns `null`. You should check if the fruit is already in the storage and handle the case when it's not.